### ISSUE TITLE
Depth of field issue

### ISSUE THREAD

**Issue by Pixelityyy** [ReplayMod/ReplayMod#403]
In the replay viewer, the depth of field option of the shaders has a problem : the focus point is locked at a certain distance. The problem is still here after rendering.
This problem appeared from the 2.4.1 version and occurs in all the next versions. No problem in 2.4.0 and before.

**Comment by Danbearpig82** [comment]
Confirmed issue with 1.16.4 and 2.5.0. It appears that sometimes spectating a player will reset the focus point to where that player is, however it never moves with the camera but stays at that point. I am testing with BSL/Complimentary Shaders.

**Comment by Pixelityyy** [comment]
Yep, and I also tested with other shaders (bbepc, sildurs vibrant...) and it doesn't fix the problem.

**Comment by Johni0702** [comment]
The only thing that changed between 2.4.0 and 2.4.1 which seems somewhat relevant is that we prevent Optifine from rendering the hands in 360 mode and while not spectating: https://github.com/ReplayMod/ReplayMod/compare/2ab92f..621587c#diff-e9833ce29337277443d1ccdbd5c24f97b54222ff2929bbd5416f98712dc555da

That doesn't seem directly related though, unless for some reason Optifine does depth of field in the `renderHand` method.
Optifine is proprietary though, so I'm neither allowed to look at its source code nor to fix it. You're at the mercy of its author.

**Comment by Pixelityyy** [comment]
Ok, thanks for your answer.
